### ISSUE TITLE
Prompt for username and password for org is not coming

### ISSUE THREAD

**Issue by sumit403114** [celador/ForceCode#367]
Issue Type: <b>Bug</b>

1). Creating a force.com project
2). Prompt for New org will appear. Selected New Org.
3). Prompt for Sandox, production, custom will appear. Selected Sandox.
4). Prompt for deploying and commiting code will appear. Selected No.
5). Prompt for username and password is not appearing to login in org.

Extension version: 3.22.5
VS Code version: Code 1.43.1 (fe22a9645b44368865c0ba92e2fb881ff1afce94, 2020-03-18T07:01:20.184Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7300U CPU @ 2.60GHz (4 x 2712)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.86GB (1.16GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by sumit403114** [comment]
![image](https://user-images.githubusercontent.com/11679461/77221421-a448d580-6b6f-11ea-98ff-0feae3f41ffe.png)

Few threads where asking to check the port number for my issue that it should run on 1717 port number. My node application is running on 1717.

**Comment by daishi4u** [comment]
Yes, this uses sfdx for login, so look-up how to change the port sfdx uses and it should then work for you. To test you can try and login via sfdx cli commands.

**Comment by nrapoza** [comment]
I am experiencing the same error, right after I upgraded Visual Studio Code to version 1.43.1
Version: 1.43.1 (system setup)
Commit: fe22a9645b44368865c0ba92e2fb881ff1afce94
Date: 2020-03-18T07:01:20.184Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.17763

1)Open an existing force.com project
2). Prompt for New org/saved org will appear. Selected saved Org.
3). ForceCode: Executing 1 Task(s) message appears
4). No prompt to log into the org

**Comment by richajain1989** [comment]
I have the exact same issue where I don't get a prompt to login to the org. I tried uninstalling and re-installing CLI but that didn't help. 
I read somewhere a .forcecode folder should get created automatically with settings.json file, but there is no such folder created for me.
Can someone please help?
